spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -48,35 +54,75 @@
           post :create, params: { recipe: { name: '', content: '' } }
         end.not_to change(Recipe, :count)
       end
+      it 'doesn\'t create recipe with level out of range' do
+        expect do
+          post :create, params: { recipe: { name: 'Leczo', content: 'Very good dish', level: 8 } }
+        end.not_to change(Recipe, :count)
+      end
```

Comment:
Try working with Rspec in this way ->
```suggestion
      let(:call) do
        post :create, params: { recipe: { name: name, content: content, level: level } }
      end
      
      context 'doesn\'t create recipe with level out of range' do
        let(:name) { 'Leczo' }
        let(:content) { 'Very good dish' }
        let(:level) { 8 }
        
        it do
          expect { call }.not_to change(Recipe, :count)
        end
      end
```